src/main/java/com/parasoft/demoapp/messages/OrderMessages.java

```diff
@@ -17,7 +17,11 @@ public class OrderMessages extends Messages {
 	public static final String REGION_CANNOT_BE_NULL = "Region should not be null.";
 	public static final String THERE_IS_NO_ORDER_CORRESPONDING_TO = "There is no order corresponding to {0}.";
 	public static final String ORDER_STATUS_CHANGED = "Order status is changed.";
+	public static final String ORDER_STATUS_CHANGED_BACK_ERROR = "Forbid changing order status from {0} to {1}, {1} status is used by Internal procedures.";
```

Comment:
Please make the message more concise like:

- Update to cancelled order is not allowed.

- Update to submitted order is only allowed in internal process.

- Invalid update for order status from {0} to {1}.